protocol/testutil/constants/prices.go

```diff
@@ -51,28 +51,33 @@ const (
 	ErrorMsgInvalidMinPriceChange   = "Min price change in parts-per-million must be greater than 0 and less than 10000"
 )
 
+var TestMarketExchangeConfigs = constants.GenerateExchangeConfigJson(constants.StaticExchangeMarketConfig)
```

Comment:
The StaticExchangeMarketConfig is actually being removed.  Would you mind creating some test constants for the exchange configs you use below instead?

Reply:
`TestMarketExchangeConfigs` was moved from line 140, and was previously used by other constants in this file as well. I made this variable public so my test case could use this file. 

You probably have a better idea than me on what's the best way to create these constants (for example, perhaps we should just move `StaticExchangeMarketConfig` to a test util file.

For now, I created raw `ExchangeConfigJson` strings generated from `StaticExchangeMarketConfig` for `BTC`, `ETH` and `SOL` and use it to replace existing usage.

Reply:
Ah, I see! I think raw string constants are fine for now.